themes/bootstrap3/templates/content/privacy.phtml

```diff
@@ -0,0 +1,7 @@
+<?php $this->headTitle($this->translate('Privacy Policy')); ?>
```

Comment:
I noticed that this file slipped in as part of a seemingly-unrelated commit. Was that intentional?

Reply:
Should have been a part of another commit, but having a template for privacy policy and link to it from the default texts was intentional. If you feel that's too much here, I'll just leave them out, but having it would help with Mink tests too.

Reply:
No objection, just wanted to be sure it was intentional and not an orphaned file. As long as there's a link to it somewhere, it's fine with me. We should probably make a note in an appropriate place to document the need for users to populate this if they want to show a privacy policy (if that hasn't already been done). It might also be valuable to have a setting that lets the privacy policy link to an external URL if desired (but maybe that's easily achievable through template customization, which would be fine too).

Reply:
We should probably have something like a "Checklist for Production Use" wiki page. We have a lot of different topics, but I'm not sure if there's a good place where to put all the things you should verify before switch to production. A typical place for a link to privacy policy would be the footer, and I suppose that's one the templates that would be modified in most cases anyway.

Reply:
Yes, a production checklist is a good idea -- and something that we could link to as a last step from all the installation notes pages. Is it worth adding a TODO to this PR to create a "production notes" section on the privacy policy wiki page (which I assume we will eventually create), and then once that is built, we can create a higher-level page to link to that along with other relevant production notes?